server/templates/control-panel.html

```diff
@@ -0,0 +1,158 @@
+<control-panel
+  v-bind:show-plots="showPlots"
+  v-bind:path-names="pathNames"
+  v-bind:map="map"
+  inline-template
+>
+  <div class="row">
+    <div class="col-sm-3">
+      <h1>Acorn control</h1>
+    </div>
+
+    <div class="col-md-5">
+      <div class="input-group" id="dropdown-container">
+        <button
+          type="button"
+          class="btn btn-info mr-1"
+          @click="toggleCollapsePlots"
```

Comment:
This equals to `v-on:click`, which calls the corresponding methods defined in `components.js`.